### ISSUE TITLE
[Bug] Star rating in addons screen is confusing

### ISSUE THREAD

**Issue by eeejay** [mozilla-mobile/fenix#8367]
## Steps to reproduce

1. Go to addons screen with TalkBack on.
1. Swipe to first addon (uBlock)
1. Swipe right again

### Expected behavior

There should be a single item with the addon name, description, rating, and user count.

### Actual behavior

There are two items, the first is the addon name, description and user count.
The second is the star rating presented as a progress bar.

![image](https://user-images.githubusercontent.com/5764/74387457-6bb61f00-4dad-11ea-9d7a-752aad79d1a4.png)

![image](https://user-images.githubusercontent.com/5764/74387468-75d81d80-4dad-11ea-8348-dc78ce3d9bbb.png)

**Comment by psymoon** [comment]
@eeejay I landed a patch in a-c for this and should be available in the latest snapshot. Can you please rebase to the latest master and verify this?

**Comment by psymoon** [comment]
@eeejay Please feel free to reopen and ping me if this is still happening. The fix should already be in Fenix nightly.

**Comment by eeejay** [comment]
Verified. Thank you!
